pennylane/measurements/measurements.py

```diff
@@ -314,6 +315,8 @@ def wires(self):
         This is the union of all the Wires objects of the measurement.
         """
         if self.mv is not None:
+            if isinstance(self.mv, Sequence):
```

Comment:
```suggestion
            if isinstance(self.mv, List):
```

Looks like `self.mv` is standardized to being `List` in initialization, so we can use that more specific value in places like here.